web/src/pages/Project.tsx

```diff
@@ -286,6 +286,14 @@ export const ProjectPage: FC = () => {
                             <Dropdown.Menu className="border border-dark shadow-lg">
                               <Dropdown.Header>
                                 <span className='text-danger'>Your PEP is invalid against {projectInfo?.pep_schema}</span>
+                                <p className="mb-0">
+                                  <span className='text-danger'>Errors found in {validationResult?.error_type}{' '}</span>
+                                  {validationResult?.sample_names && (
+                                    <OverlayTrigger overlay={<Tooltip id="validation">{validationResult?.sample_names}</Tooltip>}>
```

Comment:
Same here, maybe we could add a thing that says "samples found invalid:" Just an idea.